lua/doom/core/functions/init.lua

```diff
@@ -73,6 +73,10 @@ M.load_custom_settings = function(settings_tbl, scope)
 			for var, val in pairs(settings_tbl) do
 				vim.g[var] = val
 			end
+		elseif scope == 'options' then
+		  for var, val in pairs(settings_tbl) do
+		    vim.o[var] = val
```

Comment:
We should prefer using `vim.opt` instead because `vim.opt` is a metatable that allows you to define variables of any scope (global, window, buffer, ect) so we will not need to deal with extra setup for window/buffer options